src/components/Books.js

```diff
@@ -0,0 +1,20 @@
+import { useSelector, useDispatch } from 'react-redux';
+import * as booksActions from '../redux/books/books';
+
+const Books = () => {
+  const books = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+  const { addBook } = booksActions;
+
+  return (
+    <div>
+      <h1>
+        Bookstore
+        {books}
+      </h1>
+      <button type="button" onClick={() => dispatch(addBook())}>+</button>
+    </div>
+  );
```

Comment:
- [ ] As per the project's requirement, there should be a form component to create a book object. Kindly add this one. :pray: 
- [ ] The `books` is an array and it cannot just be rendered in an HTML element. Instead it should be mapped so that each book object should be passed as a prop for a Book component. Kindly make this change. :pray:  You can refer back to the [project's requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps.md) carefully. 
```
Books - the default view
- Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet)

- Should have a form for adding a book (no functionality yet)

```